modules/veteran_verification/app/controllers/veteran_verification/v0/disability_rating_controller.rb

```diff
@@ -1,21 +1,38 @@
 # frozen_string_literal: true
 
 require_dependency 'veteran_verification/application_controller'
+require_dependency 'notary'
 
 module VeteranVerification
   module V0
     class DisabilityRatingController < ApplicationController
+      include ActionController::MimeResponds
+
+      NOTARY = VeteranVerification::Notary.new(Settings.vet_verification.key_path)
```

Comment:
Is this going to happen once per request? If so, do you think it's worth moving this instantiation into the jwt response on L26 so that we don't add this unnecessary IO to json responses?

Reply:
This only gets loaded on the first request the first time that the controller is loaded. Had the same concern about the syscall and moved the Notary instantiation up here to prevent it 🚀

Reply:
Excellent. I thought maybe rails instantiated the controller class on each request – seems I have some reading to do!

Reply:
You should be able to verify with something as simple as a `puts` if you want to see it with your own eyes! Add the line, make a couple of requests, and look for the output.